<ui version="4.0" >
 <class>PreviewWidget</class>
 <widget class="QWidget" name="PreviewWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QScrollArea" name="scrollArea" >
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>311</width>
        <height>247</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
